setup/_pattoo/packages.py

```diff
@@ -27,9 +27,7 @@ def install_missing_pip3(package, pip_dir, verbose=False):
     username = getpass.getuser()
     if username == 'root':
         shared.run_script(
-            'python3 -m pip install {0} -t {1} -U --force-reinstall'.format(
-                                                                package,
-                                                                pip_dir),
+            'python3 -m pip install {0} -t {1} -U'.format(package, pip_dir),
```

Comment:
Why did you remove `--force-reinstall` ?

Reply:
My bad, I confused  how --upgrade worked and thought it reinstalled the packages without the --force-reinstall tag
I'm going to add it back in the next commit